cypress/e2e/home.cy.ts

```diff
@@ -1,6 +1,6 @@
 describe('Dataset browser homepage', () => {
   it('loads a dataset list', () => {
-    cy.visit('/');
+    cy.visit('/en');
```

Comment:
Somehow the redirect won't work in Cypress after the upgrade of next-intl. It does work in a normal browser. I will check why this is tomorrow.